recipes/sycomore/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set owner = "lamyj" %}
```

Comment:
Can you hard code your username instead of using a variable to be more consistent with other recipes

Reply:
Done in cd25de95b